CCPA/Data Deletion Request Handling.md

```diff
@@ -0,0 +1,329 @@
+# Data Deletion Request Handling
+
+## Summary
+
+This specification provides a way for a publisher to communicate to downstream parties about a consumer's request to delete their data. 
+
+In particular, Section 1798.105(c) of the California Consumer Protection Act (CCPA) states that "[a] business that receives a verifiable consumer request from a consumer to delete the consumer's personal information [shall] . . direct any service providers to delete the consumer's personal information from their records." 
+
+In response to this directive in the CCPA, this specification offers the technical means for publishers to communicate deletion requests to their service providers. All service providers covered by IAB Privacy LLC's Limited Service Provider Agreement (LSPA) and any other providers who receive the signal for deletion requests are then enabled to comply and delete the specified consumer data.
+
+While this specification provides the technical means for compliance in cases where CCPA applies, implementations may exist to support deletion requests outside CCPA governance. 
+
+
+### Relevant Documents
+
+[Limited Service Provider Agreement](https://www.iabprivacy.com/lspa-2019-12.pdf) (12-2-19 version)
+
+
+## How it Works
+
+Any ad tech company (vendor) supporting the US Privacy framework, exposes a JavaScript resource for each service they offer. The vendor hosts a URL that provides this resource and also serves as a unique identifier for the resource. The URL is made public to a maintained LSPA signatories list and to non-signatories via the vendor's method of choice.
+
+Publishers working with any services provided by these vendors consume the exposed JavaScript and embed it on their properties where consumers can request deletion of their personal data. If data deletion is requested, all vendor javascript on the publisher property is notified. Vendors then respond by deleting the relevant personal data and signaling any affected vendors downstream.
+
+
+### Multiple Services
```

Comment:
From the blurb above in the CCPA Regulation:

> direct any service providers to delete the consumer's personal information from their records

The relationship is consumer to service **_provider_** not consumer to **_service_**.  So, therefore, no matter how many services a Vendor provides, there is only one request to be made to that Vendor and therefore no need to support multiple services or to even specify that in this technical document.